structure/0039-pr-policy.md

```diff
@@ -0,0 +1,76 @@
+
+The goals for the PR process are:
+- **Democratic** -- Features are added through consensus of the team, and not through a benevolent dictator for life (BDFL) who has total control over the project.
+- **Quick** -- Tools should be in place to allow the team to shut down bikeshedding and say good enough on features if needed. However, this doesn't mean that pull requests should be easily forced through.
+- **Hands-off** -- Every team member shouldn't have to approve every PR. The team as a whole has a responsibility to ensure PRs are reviewed in a timely manner, but no person should have a responsibility to view every change.
+
+It is **not a goal** of this PR to be fool-proof from bad actors within the Quilt technical teams abusing the system. Being on any team in Quilt requires trust, and bad actors should be removed instead of worked around.
+
+# Explanation
+## Vocab
+- **Required Approvals**: The total number of approvals needed by the team assigned to a PR, or that team's superiors.
+    - An approval is defined as an approving review using the GitHub review feature
+- **Minimum Team Approvals**: The amount of approvals that must come directly from the assigned team. When multiple teams are assigned to a PR, this number applies to each team.
+    - This prevents PRs from getting merged by Technical Leads/etc without any oversight by the responsible team
+## Acceptance Criteria
+A Pull Request is considered to have been approved when all of the following are true:
+- The Pull Request has received the needed amount of Required Approvals and Minimum Team Reviews
+- The Pull Request has no changes requested reviews by any members of the team
+- Nobody in the team has had their review requested on the Pull Request using the system on Github
+    - This includes someone requesting the review of themselves or another person
+- All other criteria defined in the team's CONTRIBUTING file have been met.
+
+## Final Comment Period (FCP)
+- Eligable to begin immediately after the Acceptance Criteria are met.
+- Formally begins when labeled and a comment is posted with the end date of the FCP.
+- Lasts a certain amount of days, depending on the category the PR was assigned to.
+- May be ended or restarted at any time by a team member or the PR author if they have legitimate and substantiated concerns.
+
+## Other information
+Team members may dismiss their own reviews.
+
+Admins and technical leads may dismiss outstanding and requested reviews at their discretion, but should only do so if the team member cannot be contacted within a reasonable time-frame and must leave a comment on the PR with their justification. Reviews should *not* be dismissed simply because the majority (or that specific admin/technical lead) disagrees with one person or feels their concerns are invalid. Instead, a formal vote to merge a PR should be used.
+
+## Template for CONTRIBUTING.MD
+```md
```

Comment:
The proper language identifier here for Markdown is `markdown`, not `md` (it seems to be for some sort of LISP-related language from my small investigation).